scripts/tools/temporary/factory_data_template.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
If we are going to support mandatory and non-mandatory data maybe we should express it more it the json, like e.g. simply having two separate keys "mandatory_data" and "additional_data".

Reply:
As a follow up suggestion, we could  provide a json schema and validate the input json against it. I think it would be a kind of self-documenting what is actually expected as an input.